.github/workflows/tests.yml

```diff
@@ -523,3 +523,34 @@ jobs:
           dbOptions: '--dbclient=sqlite --dbfile=./tmp/data.db'
           runEE: true
           jestOptions: '--shard=${{ matrix.shard }}'
+
+  test_result:
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    name: aggregate_test_result
+    needs:
+      [
+        pretty,
+        lint,
+        build,
+        typescript,
+        unit_back,
+        unit_front,
+        e2e_ce,
+        e2e_ee,
+        cli,
+        api_ce_pg,
+        api_ce_mysql,
+        api_ce_mysql_5,
```

Comment:
Why don't we run the mysql_5 on ee too by the way?

Reply:
I don't know. considering we have dropped it in v5 and it will soon become the main branch I don't think it's worth doing anything about that :)